mac

```diff
@@ -137,45 +94,45 @@ if brew list | grep -Fq brew-cask; then
 fi
 
 fancy_echo "Updating Homebrew formulae ..."
-brew tap thoughtbot/formulae
-
 brew update
-
-fancy_echo "Updating Unix tools ..."
-brew_install_or_upgrade 'ctags'
-brew_install_or_upgrade 'git'
-brew_install_or_upgrade 'openssl'
-brew_install_or_upgrade 'rcm'
-brew_install_or_upgrade 'reattach-to-user-namespace'
-brew_install_or_upgrade 'the_silver_searcher'
-brew_install_or_upgrade 'tmux'
-brew_install_or_upgrade 'vim'
-brew_install_or_upgrade 'zsh'
-
-fancy_echo "Updating Heroku tools ..."
-brew_install_or_upgrade 'heroku-toolbelt'
-brew_install_or_upgrade 'parity'
-
-fancy_echo "Updating GitHub tools ..."
-brew_install_or_upgrade 'hub'
-
-fancy_echo "Updating image tools ..."
-brew_install_or_upgrade 'imagemagick'
-
-fancy_echo "Updating testing tools ..."
-brew_install_or_upgrade 'qt'
-
-fancy_echo "Updating programming languages ..."
-brew_install_or_upgrade 'libyaml' # should come after openssl
-brew_install_or_upgrade 'node'
-brew_install_or_upgrade 'rbenv'
-brew_install_or_upgrade 'ruby-build'
-
-fancy_echo "Updating databases ..."
-brew_install_or_upgrade 'postgres'
-brew_install_or_upgrade 'redis'
-brew_launchctl_restart 'postgresql'
-brew_launchctl_restart 'redis'
+brew bundle --file=- <<EOF
+tap "thoughtbot/formulae"
```

Comment:
The extreme pedant in me winced at the use of double-quotes in this `Brewfile` and single-quotes in the other :wink: